app_test_suite/steps/scenarios/upgrade.py

```diff
@@ -281,3 +317,32 @@ def _get_test_exec_info(self, chart_path: str, chart_ver: str, chart_config_file
             test_type=self.test_provided,
         )
         return exec_info
+
+    def _save_metadata(
+        self,
+        app_name: str,
+        chart_version: str,
+        app_version: str,
+        stable_chart_version: str,
+        stable_app_version: str,
+        cluster_type: str,
+        cluster_version: str,
+    ) -> None:
+        metadata = {
+            "appName": app_name,
+            "chartVersion": chart_version,
```

Comment:
The name is longer but I think upstreamChartVersion would be clearer here.

Reply:
But it's not upstream, it's the chart's under the test.